inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/consume/ConsumePulsarOperator.java

```diff
@@ -103,7 +109,22 @@ public InlongConsumeInfo getFromEntity(InlongConsumeEntity entity) {
             ConsumePulsarDTO dto = ConsumePulsarDTO.getFromJson(entity.getExtParams());
             CommonBeanUtils.copyProperties(dto, consumeInfo);
         }
-
+        String groupId = entity.getInlongGroupId();
+        InlongGroupInfo groupInfo = groupService.get(groupId);
+        String clusterTag = groupInfo.getInlongClusterTag();
+        List<ClusterInfo> clusterInfos = clusterService.listByTagAndType(clusterTag, ClusterType.PULSAR);
+        StringBuilder adminUrls = new StringBuilder();
+        StringBuilder serverUrls = new StringBuilder();
+        String topic = entity.getTopic();
+        Preconditions.checkNotEmpty(clusterInfos, "pulsar cluster not exist for groupId=" + groupId);
+        for (ClusterInfo clusterInfo : clusterInfos) {
+            PulsarClusterInfo pulsarCluster = (PulsarClusterInfo) clusterInfo;
+            adminUrls.append(pulsarCluster.getAdminUrl()).append(InlongConstants.SEMICOLON);
```

Comment:
Maybe these clusters are not the same cluster.
For example, cluster A and cluster B do not communicate. If their admin urls and service urls are spliced together, problems will occur.